src/main/java/frc/robot/commands/ShooterPressed.java

```diff
@@ -71,9 +71,23 @@ public void end(boolean interrupted) {
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
+
     if (i >= 50) { // 1000 miliseconds delay TODO: Use a CDS method for this when possible
       return true;
     }
+
+    if(m_ShooterSubsystem.wheelReady()){
+      SmartDashboard.putBoolean("wheelReady", true);
+      //if(i > 0 || m_LimelightSubsystem.calculatePID() == 0.0){
+        m_ShooterSubsystem.runCargo(true,true);
+        m_CDSSubsystem.CDSWheelToggle(false);
+        m_CDSSubsystem.CDSBeltToggle(false);
+        i++;
+        if(i==100){ //Expected to add a 2000ms delay
+          return true;
+       //}
+        }
+    }
```

Comment:
Merge main one more time -- ShooterPressed doesn't call the CDS directly anymore (will rely on your default command to get the balls into position)